wifiphisher/common/tui.py

```diff
@@ -939,12 +941,15 @@ def display_info(self, screen, info):
                 screen.addstr(raw_num, 0, client)
                 raw_num += 1
         try:
-            # print the dhcp lease section
+
+            # Print the dhcp lease section
             screen.addstr(7, 0, "DHCP Leases: ", self.blue_text)
-            if os.path.isfile('/var/lib/misc/dnsmasq.leases'):
-                dnsmasq_output = check_output(
-                    ['head', '-5', '/var/lib/misc/dnsmasq.leases'])
-                screen.addstr(8, 0, dnsmasq_output)
+            victims_instance = victim.Victims.getInstance()
+            dict_string = ""
+            for key in victims_instance.victims_dic.keys():
+                dict_string = key + "\t" + victims_instance.victims_dic[key].ip_address + "\t" + victims_instance.victims_dic[key].os + "\t" + victims_instance.victims_dic[key].vendor
```

Comment:
An idea here is to add a new method in the `Victim` class, called `get_print_representation` that should return the concatenated string for you.